buildkite/build_project.sh

```diff
@@ -57,8 +57,18 @@ case "$REPO_FULL_NAME" in
             "$DIR/merge_head.sh"
             latest_tag="IS-ALREADY-CHECKED-OUT"
         else
-            # otherwise checkout the respective branch
-            latest_tag=$("$DIR/origin_target_branch.sh" "${REPO_URL}")
+            # for the main core repositories,
+            # clone_repositories.sh will clone them together
+            case "$REPO_FULL_NAME" in
+                "dlang/dmd" | \
+                "dlang/druntime" | \
+                "dlang/phobos")
```

Comment:
tools and dub as well?

Reply:
No they are checked out as individual repos as they don't depend on the core repos - just an up to date DMD which is always built initially and already provided ty them.

Reply:
This is just about these three repos as for them clone_repositories will be executed during "project testing"

Reply:
Ok.